redfish-core/lib/systems.hpp

```diff
@@ -1415,17 +1415,6 @@ inline void getTrustedModuleRequiredToBoot(
             return;
         }
 
-        /* When there is more than one TPMEnable object... */
-        if (subtree.size() > 1)
```

Comment:
instead of commenting everything just disble internal error  return and update log message syaying  required to enable fduring multihost TPM enbalemnet.